GitCommands/Settings/AppSettings.cs

```diff
@@ -615,7 +615,7 @@ public static bool DontConfirmAmend
 
         public static bool DontConfirmPushNewBranch
         {
-            get { return GetBool("DontConfirmPushNewBranch", false); }
+            get { return GetBool("DontConfirmPushNewBranch", true); }
```

Comment:
> Should it be moved from the Advanced to the Detailed tab?

What is the difference between `Advanced` and `Detailed`?

Anyway it's nothing to do with my changes in this PR.